### DISCUSSION TITLE
I would like to test Elastic Endpoint Security

### DISCUSSION THREAD

**Discussion by tlinsenmeyer** [Security-Onion-Solutions/securityonion#2251] [Unsupported Versions]
In my home lab, I would like to test Elastic Endpoint Security on 3 endpoints.  This is a basic license feature, however it requires an encrypted connection between kibana and elastisearch.  Any help to accomplish this without break SO would be awesome.

**Comment by weslambert** [comment]
We don't officially support integration at this time, but I'll try having a look at as I get time.  I believe this would tie in with Elastic Auth, which is on the roadmap for this version of Security Onion, but not yet tested/implemented.
